segbot_sensors/CMakeLists.txt

```diff
@@ -82,3 +86,9 @@ if (CATKIN_ENABLE_TESTING)
   roslaunch_add_file_check(launch/kinect)
   roslaunch_add_file_check(launch/ptgrey)
 endif()
+
+add_executable(batteryDiagnostics src/segbot_sensors/battery_monitor.cpp)
+target_link_libraries(batteryDiagnostics ${catkin_LIBRARIES})
+
+add_executable(voltagePub src/segbot_sensors/fakeVoltPub.cpp)
+target_link_libraries(voltagePub ${catkin_LIBRARIES})
```

Comment:
You also need a RUNTIME DESTINATION. See http://wiki.ros.org/catkin/CMakeLists.txt#Optional_Step:_Specifying_Installable_Targets for the example. You can have all your targets in a single install command.